### PR TITLE
Add MixinStakingPool unit tests.

### PR DESCRIPTION
**Pull request by dorothy-zbornak**
## Description

`MixinStakingPool` unit tests in the hizzy! :house: 

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: contracts/test-utils/src/log_utils.ts [hunk 1]
```diff
@@ -32,3 +32,18 @@ export function verifyEvents<TEventArgs>(
         expect(log).to.deep.equal(expectedEvents[index]);
     });
 }
+
+/**
+ * Given a collection of logs, verifies that matching events are identical.
+ */
+export function verifyEventsFromLogs<TEventArgs>(
```
Comment (mzhu25) [inline contracts/test-utils/src/log_utils.ts]: Can we get rid of either this or `verifyEvents` now that the latter has been merged?

File: contracts/staking/test/unit_tests/staking_pool_test.ts [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+    TestMixinStakingPoolContract,
+    TestMixinStakingPoolEvents,
+    TestMixinStakingPoolStakingPoolCreatedEventArgs as StakingPoolCreated,
+} from '../../src';
+
+blockchainTests.resets('MixinStakingPool unit tests', env => {
+    let testContract: TestMixinStakingPoolContract;
+    let operator: string;
+    let maker: string;
+    let notOperatorOrMaker: string;
```
Comment (mzhu25) [inline contracts/staking/test/unit_tests/staking_pool_test.ts]:
```suggestion
    let pleb: string;
```
70% jk

File: contracts/staking/test/unit_tests/staking_pool_test.ts [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+                from: maker,
+            });
+            verifyEventsFromLogs(
+                logs,
+                [
+                    {
+                        makerAddress: maker,
+                        poolId,
+                    },
+                ],
```
Comment (mzhu25) [inline contracts/staking/test/unit_tests/staking_pool_test.ts]: ew prettier wtf

File: contracts/test-utils/src/log_utils.ts (hunk 1 above)
Comment (dorothy-zbornak) [reply contracts/test-utils/src/log_utils.ts]: Decided to just refactor `verifyEvents()` to use `verifyEventsFromLogs()` so they can coexist without duping code.

**Pull request merged**